html/join.html

```diff
@@ -7,57 +7,118 @@
+            <div>
+              <label class="form-label">Username
+                <input class="username" id="username" name="username" type="text" required>
+              </label>
+            </div>
+            <div>
+              <label class="form-label">Password
+                <input class="password" id="password" name="password" type="password" disabled>
+              </label>
+            </div>
+          </div>
+          <div class="user-inputs-bottom-row">
+            <div class="flex-column">
+              <label class="form-label">Role
+                <select id="role-select" class="role-select" disabled></select>
+              </label>
+            </div>
+            <input id="join-button" class="join-button" type="submit" value="Join" disabled>
           </div>
         </form>
-      </div>
 
-      <div id="session-list-container" class="flex-column session-list-container">
-        <input id="session-filter-box" type="search" placeholder="Filter...">
-        <div id="session-option-container" class="flex-column"></div>
-        <template id="session-option-template">
-          <label class="session-label">
-            <span class="session-name"></span> | <span class="session-model-name"></span> | <span class="session-info"></span> connected
-            <input class="session-option" name="sessionOption" type="radio">
-          </label>
-        </template>
       </div>
 
+      <div class="right-column">
+        <img
+          id="session-preview-image"
+          class="session-preview-image"
+          src="http://ccl.northwestern.edu/netlogo/models/models/HubNet%20Activities/Disease%20HubNet.png"
+          draggable=false
+        />
+
+        <div id="chat-box-closed" class="chat-box-closed">
+          <div class="chat-box-closed-header">
+            <div>Chat</div>
+            <div class="expand-chat-arrow"></div>
+          </div>
+        </div>
+
+        <div id="chat-box-open" class="chat-box-open">
+          <div id="open-chat-header" class="open-chat-header">
+            <div>Chat</div>
+            <div class="collapse-chat-arrow"></div>
+          </div>
+          <div>
+            <div id="global-chat">
+              <div id="global-chat-output" class="open-chat-body"></div>
+              <input
+                id="global-chat-input"
+                class="open-chat-input"
+                type="text"
+                value=""
+                placeholder="Type chat here..."
+              />
+            </div>
+            <div id="session-chat" class="hidden">
+              <div id="session-chat-output" class="open-chat-body"></div>
+              <input
+                id="session-chat-input"
+                class="open-chat-input"
+                type="text"
+                value=""
+                placeholder="Type chat here..."
+              />
+            </div>
+          </div>
+        </div>
+
     </div>
 
     <div id="nlw-frame" class="frame hidden">
```

Comment:
Somehow, your changes caused `nlw-frame` to end up inside of `session-browser-frame`.  As a result, when "Join" is clicked, instead of proceeding to the model, we just get a white page, because `nlw-frame` is inside the now-hidden browser frame.